pkg/function/kube_task.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/kanisterio/kanister/pkg/output"
 	"github.com/kanisterio/kanister/pkg/param"
 	"github.com/kanisterio/kanister/pkg/progress"
+	"strings"
```

Comment:
This should be a part of the standard package group at the top.
@mabhi if you use VSCode, you can set it up to auto-correct this on save.

Reply:
Done